server/gitrest/.prettierignore

```diff
@@ -32,16 +27,10 @@ tools/pipelines/
 # Generated bundle analysis files
 **/bundleAnalysis/**
 
-# Generated oclif manifest files
-**/oclif.manifest.json
```

Comment:
I think these should stay ignored. They are auto-generated files.

Reply:
I don't think there are any in this directory: we only use it in tools and build tools